src/components/stepforms/DissolveStepForm.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <StepFormHeader
+      :title="title"
+      :stepName="editedStep.name"
+      :version="version"
+      :backendError="backendError"
+    />
+    <MultiselectWidget
+      class="groupbyColumnsInput"
+      v-model="editedStep.groups"
+      name="Group rows by..."
+      :options="columnNames"
+      placeholder="Add columns"
+      data-path=".groups"
+      :errors="errors"
+      :available-variables="availableVariables"
+      :variable-delimiters="variableDelimiters"
+    />
+    <ListWidget
+      v-model="editedStep.aggregations"
+      addFieldName="Add aggregation"
+      class="toremove"
+      name="And aggregate..."
+      :defaultItem="defaultAggregation"
+      :widget="widgetAggregation"
+      :automatic-new-field="false"
+      data-path=".aggregations"
+      :errors="errors"
+      :available-variables="availableVariables"
+      :variable-delimiters="variableDelimiters"
+    />
+    <CheckboxWidget
+      class="keepNullsCheckbox"
+      label="Include null values in results"
+      v-model="editedStep.include_nulls"
```

Comment:
It's been a while since we made a new step so I forgot how we usually handle case in there. Do we have something that does snake to camel somewhere? Or do we keep snake in the front-end too in other steps?

Reply:
I think it's up to the pydantic models to provide a camel case alias, I'll update `DissolveStep`